src/index.html

```diff
@@ -15,8 +15,79 @@
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+
+    />
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <nav>
+        <ul>
+          <img
+            class="img"
+            src="/src/images/logo.png"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. Therefore, 'img' is not a good class name. Please, provide a more meaningful class name. For example, if it's a company logo, you can write 'logo'.